opendevin/controller/agent_controller.py

```diff
@@ -47,6 +47,7 @@ class AgentController:
     event_stream: EventStream
     state: State
     agent_task: Optional[asyncio.Task] = None
+    parent: 'AgentController | None' = None
```

Comment:
```suggestion
    parent: AgentController | None = None
```
just trivial detail, the class is obviously available, it seems like there shouldn't be a need to hide it

Reply:
Ruff is unhappy about that:

```
ruff.....................................................................Failed
- hook id: ruff
- exit code: 1

opendevin/controller/agent_controller.py:51:15: F821 Undefined name `AgentController`
```

Reply:
Hah, okay.